notebooks/ldb_Feature Engineering_kaggleEX.ipynb

```diff
@@ -0,0 +1,2643 @@
+{
```

Comment:
<p>I think the same error of the other notebook occurred here. You forgot to define this full_pipeline atribute somewhere. If I understood it correctly, it should be a combination of some data transforming, right?</p><p> </p><p>Furthermore, I believe the feedback of explaining the code in markdowns instead of commenting the code is also valid here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>